URL Detection

```diff
@@ -0,0 +1,3 @@
+NetworkCommunicationEvents  
+| where RemoteUrl == "www.microsoft.com"
+| project EventTime , ComputerName , MachineId , ReportId 
```

Comment:
Sorry about nitpicking, but please remove the extra spaces before column names (mainly for readability):
`| project EventTime, ComputerName, MachineId, ReportId`